lib/BaseStrategy.ts

```diff
@@ -80,7 +80,7 @@ export abstract class BaseStrategy extends Strategy {
     if (token) {
       const tokenPayload = this.decodeToken(token);
       this.validate(token, tokenPayload, verified);
-    } else if (siweObject.signature && siweObject.message) {
+    } else if (siweObject && siweObject.signature && siweObject.message) {
```

Comment:
If `siweObject` is not null, then it has `signature` and `message`
```suggestion
    } else if (siweObject) {
```